src/styles/main.scss

```diff
@@ -1,3 +1,39 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/functions";
+
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/product.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
-  margin: 0;
+  margin: 0 auto;
+  max-width: getWith(4, 250px);
+}
+
+.cards-container {
+  padding: 51px 0;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+@media (max-width: 1024px) {
```

Comment:
You need to have always 4 cards in a row
<img width="527" alt="Screenshot 2023-04-07 at 22 04 07" src="https://user-images.githubusercontent.com/117587463/230663399-7292babb-0edc-4109-afa4-d97e50efc8d7.png">

Reply:
The way to fix it: set width for `cards-container` - then a horizontal scroll will appear but cards will not be wrapped